community/mei-board.md

```diff
@@ -22,21 +28,16 @@ The MEI Board manages the property and business of MEI. It promotes the developm
 * **Perry Roland**, University of Virginia Library, Charlottesville (previous terms 2015–2018)
 * **Martha Thomae**, McGill University, Montréal
 
-### Term 2019-2021
-
-* **Benjamin W. Bohl (Technical Co-Chair)**, Institut für Musikwissenschaft, Goethe Universität Frankfurt, Frankfurt am Main and Musikwissenschaftliches Seminar Detmold/Paderborn
-* **Elsa De Luca (Administrative Chair)**, Centro de Estudos de Sociologia e Estética Musical (CESEM), Universidade Nova de Lisboa, Lisboa
-* **Ichiro Fujinaga**, McGill University, Montréal
-
-
-
 ### Previous Board Members
 
-* **Nobert Dubowy**, Internationale Stiftung Mozarteum, Salzburg (2018-2020)
+* **Nobert Dubowy**, Internationale Stiftung Mozarteum, Salzburg (2018-2020)
 * **Andrew Hankinson**, Bodleian Library, Oxford (2017–2019)
 * **Eleanor Selfridge-Field**, Center for Computer-Assisted Research in the Humanities, Stanford University (2015–2019)
-* **Giuliano Di Bacco**, Jacobs School of Music, Indiana University, Bloomington, (2015-2017)
-* **Axel Teich Geertinger**, Danish Centre for Music Editing, Royal Library, Copenhagen, (2015-2016)
+* **Giuliano Di Bacco**, Jacobs School of Music, Indiana University, Bloomington, (2015-2017)
+* **Axel Teich Geertinger**, Danish Centre for Music Editing, Royal Library, Copenhagen, (2015-2016)
+* **Elsa De Luca**, Centro de Estudos de Sociologia e Estética Musical (CESEM), Universidade Nova de Lisboa, Lisboa (2019–2021)
+* **Ichiro Fujinaga**, McGill University, Montréal (2015–2021)
```

Comment:
This should be sorted either alphabetically (De Luca, Di Bacco, Dubowy...) or by date in descending order. If sorted by date, it could be good to prioritize the end date.

Reply:
I think alphabetical order is better here, because the dates are at the end.